src/ckeditor.js

```diff
@@ -5,67 +5,77 @@
-import ImagecaptionPlugin from '@ckeditor/ckeditor5-image/src/imagecaption';
-import ImagestylePlugin from '@ckeditor/ckeditor5-image/src/imagestyle';
-import ImagetoolbarPlugin from '@ckeditor/ckeditor5-image/src/imagetoolbar';
-import ImageuploadPlugin from '@ckeditor/ckeditor5-image/src/imageupload';
+import ImageCaptionPlugin from '@ckeditor/ckeditor5-image/src/imagecaption';
+import ImageStylePlugin from '@ckeditor/ckeditor5-image/src/imagestyle';
+import ImageToolbarPlugin from '@ckeditor/ckeditor5-image/src/imagetoolbar';
+import ImageUploadPlugin from '@ckeditor/ckeditor5-image/src/imageupload';
 import LinkPlugin from '@ckeditor/ckeditor5-link/src/link';
 import ListPlugin from '@ckeditor/ckeditor5-list/src/list';
 import ParagraphPlugin from '@ckeditor/ckeditor5-paragraph/src/paragraph';
 
 export default class ClassicEditor extends ClassicEditorBase {}
 
-ClassicEditor.build = {
-	plugins: [
-		EssentialsPlugin,
-		UploadadapterPlugin,
-		AutoformatPlugin,
-		BoldPlugin,
-		ItalicPlugin,
-		BlockquotePlugin,
-		EasyimagePlugin,
-		HeadingPlugin,
-		ImagePlugin,
-		ImagecaptionPlugin,
-		ImagestylePlugin,
-		ImagetoolbarPlugin,
-		ImageuploadPlugin,
-		LinkPlugin,
-		ListPlugin,
-		ParagraphPlugin
-	],
-	config: {
-		toolbar: {
-			items: [
-				'heading',
-				'|',
-				'bold',
-				'italic',
-				'link',
-				'bulletedList',
-				'numberedList',
-				'imageUpload',
-				'blockQuote',
-				'undo',
-				'redo'
-			]
-		},
-		image: {
-			toolbar: [
-				'imageStyle:full',
-				'imageStyle:side',
-				'|',
-				'imageTextAlternative'
-			]
-		},
-		language: 'en'
-	}
+// Plugins which will be automatically loaded when creating an instance of this editor build.
+// Note: New plugins should not be added to an existing editor build. They should be added at this stage.
+ClassicEditor.defaultPlugins = [
+	EssentialsPlugin,
+	UploadAdapterPlugin,
+	AutoformatPlugin,
+	BoldPlugin,
+	ItalicPlugin,
+	BlockQuotePlugin,
+	EasyImagePlugin,
+	HeadingPlugin,
+	ImagePlugin,
+	ImageCaptionPlugin,
+	ImageStylePlugin,
+	ImageToolbarPlugin,
+	ImageUploadPlugin,
+	LinkPlugin,
+	ListPlugin,
+	ParagraphPlugin
+];
+
+// The default configuration of this editor build.
+// It can be overridden by passing your own options to the `create()` method.
+//
+// See all the configuration options:
+// https://docs.ckeditor.com/ckeditor5/latest/api/module_core_editor_editorconfig-EditorConfig.html
+ClassicEditor.defaultConfig = {
+	toolbar: {
+		items: [
+			'heading',
+			'|',
+			'bold',
+			'italic',
+			'link',
+			'bulletedList',
+			'numberedList',
+			'imageUpload',
+			'blockQuote',
+			'undo',
+			'redo'
+		]
+	},
+	image: {
+		toolbar: [
+			'imageStyle:full',
+			'imageStyle:side',
+			'|',
+			'imageTextAlternative'
+		]
+	},
+
+	// UI language. Language codes follow the https://en.wikipedia.org/wiki/ISO_639-1 format.
+	//
+	// See: https://docs.ckeditor.com/ckeditor5/latest/features/ui-language.html
+	language: 'en'
```

Comment:
If we want to deduplicate it, this variable could be also overwritten by `CKEditorWebpackPlugin` since it already replaces some strings in the editor.